kanidmd/score/src/https/mod.rs

```diff
@@ -448,6 +476,8 @@ pub fn create_https_server(
         static_tserver.with(compress_middleware.clone());
 
         static_tserver.at("/").get(index_view);
+        static_tserver.at("/robots.txt").get(robots_txt);
+        static_tserver.at("/manifest.webmanifest").get(manifest);
```

Comment:
What does a webmanifest do?

Reply:
It defines a bunch of stuff around what happens when you open the site, then click share -> "add to home screen" on your phone - sets the icon to the nice login square and the domain display name 😄

Reply:
Ooooooooooo that's freaking awesome, I'm sold.